node/src/network.rs

```diff
@@ -16,69 +18,180 @@ pub enum Network {
+                "/dnsaddr/da-full-1.celestia-bootstrap.net/p2p/12D3KooWKZCMcwGCYbL18iuw3YVpAZoyb1VBGbx9Kapsjw3soZgr",
+                "/dnsaddr/da-full-2.celestia-bootstrap.net/p2p/12D3KooWE3fmRtHgfk9DCuQFfY3H3JYEnTU3xZozv1Xmo8KWrWbK",
+                "/dnsaddr/da-full-3.celestia-bootstrap.net/p2p/12D3KooWK6Ftsd4XsWCsQZgZPNhTrE5urwmkoo5P61tGvnKmNVyv",
+            ],
+            Network::Arabica => &[
+                "/dnsaddr/da-bridge-1.celestia-arabica-11.com/p2p/12D3KooWGqwzdEqM54Dce6LXzfFr97Bnhvm6rN7KM7MFwdomfm4S",
+                "/dnsaddr/da-bridge-2.celestia-arabica-11.com/p2p/12D3KooWCMGM5eZWVfCN9ZLAViGfLUWAfXP5pCm78NFKb9jpBtua",
+                "/dnsaddr/da-bridge-3.celestia-arabica-11.com/p2p/12D3KooWEWuqrjULANpukDFGVoHW3RoeUU53Ec9t9v5cwW3MkVdQ",
+                "/dnsaddr/da-bridge-4.celestia-arabica-11.com/p2p/12D3KooWLT1ysSrD7XWSBjh7tU1HQanF5M64dHV6AuM6cYEJxMPk",
+            ],
+            Network::Mocha => &[
+                "/dnsaddr/da-bridge-mocha-4.celestia-mocha.com/p2p/12D3KooWCBAbQbJSpCpCGKzqz3rAN4ixYbc63K68zJg9aisuAajg",
+                "/dnsaddr/da-bridge-mocha-4-2.celestia-mocha.com/p2p/12D3KooWK6wJkScGQniymdWtBwBuU36n6BRXp9rCDDUD6P5gJr3G",
+                "/dnsaddr/da-full-1-mocha-4.celestia-mocha.com/p2p/12D3KooWCUHPLqQXZzpTx1x3TAsdn3vYmTNDhzg66yG8hqoxGGN8",
+                "/dnsaddr/da-full-2-mocha-4.celestia-mocha.com/p2p/12D3KooWR6SHsXPkkvhCRn6vp1RqSefgaT1X1nMNvrVjU2o3GoYy",
+            ],
+            Network::Custom(_) => &[],
+        };
+        peers
+            .iter()
+            .map(|s| s.parse().expect("Invalid bootstrap address"))
+    }
+}
 
 impl FromStr for Network {
-    type Err = UnknownNetworkError;
-
-    fn from_str(network_id: &str) -> Result<Self, Self::Err> {
-        match network_id {
-            "arabica-11" => Ok(Network::Arabica),
-            "mocha-4" => Ok(Network::Mocha),
-            "private" => Ok(Network::Private),
-            network => Err(UnknownNetworkError(network.to_string())),
+    type Err = InvalidNetworkId;
+
+    fn from_str(value: &str) -> Result<Self, InvalidNetworkId> {
+        match value {
+            "Mainnet" | "MainNet" | "mainnet" | "celestia" => Ok(Network::Mainnet),
+            "Arabica" | "arabica" | "arabica-11" => Ok(Network::Arabica),
+            "Mocha" | "mocha" | "mocha-4" => Ok(Network::Mocha),
+            custom_id => Network::custom(custom_id),
         }
     }
 }
 
-/// Get the string id of the given network.
-pub fn network_id(network: Network) -> &'static str {
-    match network {
-        Network::Arabica => "arabica-11",
-        Network::Mocha => "mocha-4",
-        Network::Private => "private",
-        Network::Mainnet => "celestia",
+impl fmt::Display for Network {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let s = match self {
+            Network::Mainnet => "mainnet",
+            Network::Arabica => "arabica",
+            Network::Mocha => "mocha",
```

Comment:
Should I capitalize them?

Reply:
Network ID is returned with `id()` method. This is on purpose because the real network ID a bit different, for example network ID of Arabica is `arabica-11` etc.

Reply:
ah okay, then idc, up to you I guess

Reply:
I capitalize them